debug.go

```diff
@@ -20,6 +20,12 @@ func (s *spanImpl) Lock() {
 }
 
 func (s *spanImpl) maybeAssertLocked() {
```

Comment:
nit: this would be clearer to me as `assertSanityLocked` or `maybeAssertSanityLocked` (to the casual reader it looks like you're asserting that spanImpl is locked)